nginx/nginx.go

```diff
@@ -191,6 +182,35 @@ func (n *nginxUpdater) initialiseNginxConf() error {
 	return err
 }
 
+func (n *nginxUpdater) ensureNginxRunning() error {
+	// Guard against the (hopefully unlikely) event that two updates could be
+	// received concurrently and attempt to start Nginx twice.
+	n.initialUpdateApplied.Lock()
+	defer n.initialUpdateApplied.Unlock()
+
+	if !n.initialUpdateApplied.done {
+		log.Info("Starting nginx with initial configuration")
```

Comment:
Can we change the comment to "Starting nginx for the first time"? As we no longer have an "initial configuration".